README.md

```diff
@@ -48,12 +42,59 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [TiDB docs](https://docs.pingcap.com/tidb/stable) and [release notes](https://docs.pingcap.com/tidb/dev/release-notes).
 
+## Community
+
+You can join these groups and chats to discuss and ask TiDB related questions:
+
+- [Contributors Mailing list](https://lists.tidb.io/g/contributors)
+- [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
+- [Chinese Forum](https://asktug.com)
+
+In addition to that, you may enjoy following:
```

Comment:
```suggestion
In addition to groups and chats above, you may enjoy following:
```
`that` is not clear about what is referenced

Reply:
I'd prefer just "In addition, you may enjoy following:"

Reply:
```suggestion
In addition, you may enjoy following:
```